Mathlib/SetTheory/Surreal/Basic.lean

```diff
@@ -358,6 +384,14 @@ instance orderedAddCommGroup : OrderedAddCommGroup Surreal where
   nsmul := nsmulRec
   zsmul := zsmulRec
 
+lemma add_def {x y: PGame} (hx: x.Numeric) (hy: y.Numeric):
```

Comment:
These lemmas are all misnamed, I think. They will not actually let you rewrites `x + y` for `x y : Surreal`, which is what they lemma with this name must do. 

Suggest flipping sides and renaming `mk_add`, etc.